.circleci/config.yml

```diff
@@ -179,7 +180,7 @@ jobs:
     working_directory: /home/circleci/tasking-manager
     resource_class: large
     docker:
-      - image: cimg/python:3.7.14-node
+      - image: cimg/python:3.10.10-node
```

Comment:
Since cimg/python:3.10.10-node users node v18 frontend build might not pass until frontend is compatible with node v18. Let's use a different images for frontend and backend jobs here.

Reply:
It looks like I can use cimg/python:3.10.7-node -- that image has node node 16.17.0.